tests/webvtt_parser.py

```diff
@@ -154,3 +154,11 @@ def test_clean_cue_tags(self):
     def test_parse_captions_with_bom(self):
         vtt = webvtt.read(self._get_file('captions_with_bom.vtt'))
         self.assertEqual(len(vtt.captions), 4)
+
+    def test_can_parse_empty_lines_from_netflix_subtitles(self):
```

Comment:
I would use a more generic and precise test name here like test_parse_captions_with_empty_lines

Reply:
I was trying to be specific so that we know why we were supporting it, since I don't believe what they had done was truly correct format of VTT, but when parsing files we'd rather try to work, even if it didn't follow the spec. 

I'll pinpoint the line that was causing the issue and make a smaller test case instead of using a whole netflix file so that its a little more generic